### DISCUSSION TITLE
Weird behaviour of MpFace in Chrome

### DISCUSSION THREAD

**Discussion by bbbbbgdn** [cables-gl/cables_docs#789] [! Welcome - readme first!]
Found rather weird behaviour:
If I set webcam texture size to 512x512 MpFace in Chrome detects face half of the time, or the first time and then never.  Changing the resolution back to default helps. 

Seems to works in all other browsers.
https://cables.gl/p/L793Gf

Does this look like a bug?

**Comment by steam0r** [comment]
can you verify this with mediapipe somehow? the op basically just calls their api/library:

it's almost exactly what they are doing in this example:
https://github.com/google-ai-edge/mediapipe/blob/master/docs/solutions/face_mesh.md
